src/Sylius/Component/Core/Dashboard/SalesDataProvider.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Dashboard;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+
+/**
+ * @experimental
+ */
+final class SalesDataProvider implements SalesDataProviderInterface
+{
+    /** @var EntityManagerInterface */
+    private $entityManager;
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    public function getLastYearSalesSummary(ChannelInterface $channel): SalesSummaryInterface
+    {
+        $startDate = (new \DateTime('first day of next month last year'))->format('Y/m/d');
+        $endDate = (new \DateTime('last day of this month'))->format('Y/m/d');
+        $channelId = $channel->getId();
+
+        $query = $this->entityManager->getConnection()->query(
+            "SELECT
+                DATE_FORMAT(checkout_completed_at, '%m.%y') AS \"date\",
+                SUM(total) as \"total\"
+            FROM sylius_order
+            WHERE (channel_id = $channelId)
```

Comment:
To support string and int ids:
```suggestion
            WHERE (channel_id = "$channelId")
```